tests/Checks/ExternalURLCheckTest.php

```diff
@@ -1,29 +0,0 @@
-<?php
```

Comment:
This test is permanently skipped, so may as well just delete it

Reply:
Did you spend any time at all looking at what's wrong with it and if it can be fixed instead?

Reply:
We should probably rework that to not use curl so we can mock it so we can test it... but that won't be agreed to as part of this PR so I'll leave it.